### ISSUE TITLE
Suport `MsgRun` param as an Account Method

### ISSUE THREAD

**Issue by leohhhn** [gnolang/gno-js-client#145]
## Description

We should support the `MsgRun` message type as an Account Method in `gno-js`.

**Comment by albttx** [comment]
Yes please 🙏🏼  this will be needed for https://github.com/gnolang/gno/pull/906 ( foo20 airdrop )
